chevah/empirical
# [#2367] Fix string listing in errors.

# Problem

When string differs due to non printable characters it is hard to see the different from the reported error.
# Changes

I have forced the usage of repr() instead of str() in AssertionError messages
# How to test

reviewers: @alibotean 

There is no automated test, but for example you can try to assert `bla\n` and 'bla\r\n` ... in the old version you will see that they differ... but the printed output look the same.

I have released the version and already used in on server in a branch and it looks ok.

Thanks!
